swim/join_sender.go

```diff
@@ -162,6 +203,15 @@ func (j *joinSender) parseHosts(hostports []string) {
 	}
 }
 
+func newDelayOpts() *delayOpts {
+	return &delayOpts{
```

Comment:
On a second read, I don't think this is useful. When I was reviewing this I thought it was in a hot code path, but this is only being run on startup.